Makefile.am

```diff
@@ -295,6 +295,47 @@ libeos_updater_flatpak_installer_libeos_updater_flatpak_installer_@EUFI_API_VERS
 # eos-updater-flatpak-installer program
 libexec_PROGRAMS += eos-updater-flatpak-installer/eos-updater-flatpak-installer
 
+###############################################################################
+# Introspection
+-include $(INTROSPECTION_MAKEFILE)
+INTROSPECTION_GIRS =
+INTROSPECTION_SCANNER_ARGS =
+INTROSPECTION_COMPILER_ARGS =
+AM_DISTCHECK_CONFIGURE_FLAGS += --enable-introspection
+EXTRA_DIST += m4/introspection.m4
+
+if HAVE_INTROSPECTION
+libeos-updater-flatpak-installer/Libeufi-@EUFI_API_VERSION@.gir: libeos-updater-flatpak-installer/libeos-updater-flatpak-installer-@EUFI_API_VERSION@.la
+libeos_updater_flatpak_installer_Libeufi_@EUFI_API_VERSION@_gir_INCLUDES = GObject-2.0 Gio-2.0
+libeos_updater_flatpak_installer_Libeufi_@EUFI_API_VERSION@_gir_CFLAGS = \
+	$(libeos_updater_flatpak_installer_libeos_updater_flatpak_installer_@EUFI_API_VERSION@_la_CFLAGS) \
+	$(NULL)
+libeos_updater_flatpak_installer_Libeufi_@EUFI_API_VERSION@_gir_FILES =
+	libeos-updater-flatpak-installer/determine-flatpak-actions.c \
+	libeos-updater-flatpak-installer/installer.h \
+	libeos-updater-flatpak-installer/perform-flatpak-actions.c \
```

Comment:
This list could just be `$(libeos_updater_flatpak_installer_libeos_updater_flatpak_installer_@EUFI_API_VERSION@_la_SOURCES)`.